.gitattributes

```diff
@@ -0,0 +1,57 @@
+# Souce: https://rehansaeed.com/gitattributes-best-practices/#line-endings
+
+###############################
+# Git Line Endings            #
+###############################
+
+# Set default behaviour to automatically normalize line endings.
+* text=auto eol=lf
+
+# Force batch scripts to always use CRLF line endings so that if a repo is accessed
+# in Windows via a file share from Linux, the scripts will work.
+
+*.bat text eol=crlf
+*.cmd text eol=crlf
+*.sln text eol=crlf
+
+# Force bash scripts to always use LF line endings so that if a repo is accessed
+# in Unix via a file share from Windows, the scripts will work.
+*.sh text eol=lf
+
+###############################
+# Git Large File System (LFS) #
+###############################
+
+*.dll binary
+*.bmp binary
+*.snk binary
+*.sql diff
+
+# Archives
+*.7z binary
+*.br binary
+*.gz binary
+*.tar binary
+*.zip binary
+
+# Excel
+*.xlsx binary
+
+# Documents
+*.pdf binary
+
+# Images
+*.gif binary
+*.ico binary
+*.jpg binary
+*.jpeg binary
+*.pdf binary
+*.png binary
+*.psd binary
+*.webp binary
+
+# Fonts
+*.woff2 binary
+
+# Other
+*.exe binary
```

Comment:
I think we can drop this. Git is smart enough to work this out and means we don't need to manage it. As your linked blog post says...

> ...relying on Git to auto-detect binary files (yes Git is clever enough to do that)

Reply:
We've seen xlsx get diffed on our repo, which is why I lost trust in the auto detect a little. Including it in the file solved that, but will make the change regardless.

Reply:
@garethbowen have made the change